docs/wiki/feature/medical-ai.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: wiki
+title: Medical AI
+description: Makes AI heal themselves and each other with the ACE Medical system.
+group: feature
+category: realism
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 7
+  patch: 0
+---
+
```

Comment:
Let's add a note here that it only works for basic medical at the moment.

```
<div class="panel callout">
    <h5>Note:</h5>
    <p>Medical AI currently only works for Basic Medical!</p>
</div>
```
Maybe a more in-depth description below on different combinations, where it works and where not (I've seen those questions already).